pkg/types/openstack/platform.go

```diff
@@ -28,6 +28,13 @@ type Platform struct {
 	// Existing Floating IP to associate with the OpenStack load balancer.
 	LbFloatingIP string `json:"lbFloatingIP"`
 
+	// ExternalDNS
+	// IP addresses of dns servers that will be added to the private
+	// networks managed by the installer
+	//
+	// This is an optional parameter, default value: empty
```

Comment:
`+optional` should be enough.

also provide information regarding which network range it needs to belong to or doesn't need to belong to. Add corresponding validation.

Reply:
I'm not sure it would need to belong to any network range, as long as the addresses provided are reachable and valid